ludwig/data/preprocessing.py

```diff
@@ -52,7 +52,7 @@
 from ludwig.utils.misc_utils import (get_from_registry, merge_dict,
                                      resolve_pointers, set_random_seed,
                                      get_proc_features_from_lists)
-from ludwig.utils.type_utils import Column
+# from ludwig.utils.type_utils import Column
```

Comment:
I was running into an issue with importing Column. Will uncomment in a subsequent patch

Reply:
Fixed in the latest patch